ui/test-view/ui/Test.jsx

```diff
@@ -82,36 +82,40 @@ BugCountComponent.propTypes = {
 // TODO: Move `Platform` into its own file.
 class Platform extends React.Component {
   getIcon(failureClassification) {
+    this.faIcon = null;
     switch (failureClassification) {
       case 'not classified':
-        return;
+        return this.faIcon;
       case 'intermittent':
-        return (
+        this.faIcon = (
```

Comment:
Please don't use `this.faIcon` and use `//eslint-disable-next-line` if moving the function outside does not work.